accounting/accounting/doctype/item/test_item.py

```diff
@@ -0,0 +1,48 @@
+# Copyright (c) 2023, Akhil and Contributors
+# See license.txt
+import random
+import string
+
+import frappe
+from frappe.tests.utils import FrappeTestCase
+
+
+class TestItem(FrappeTestCase):
+	def test_create_item_without_mandatory_fields(self):
```

Comment:
Nice tests for app. 

Note for future: Doc creation, permission etc are considered framework features. So we kinda assume they work for the most part and don't add separate tests for them.